arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -0,0 +1,366 @@
+        },
+        "eventTypesToAudit": {
+          "type": "string",
+          "description": "The events types of the Azure SQL Server auditing policy."
+        },
+        "storageAccountName": {
+          "type": "string",
+          "description": "The storage account name of the Azure SQL Server auditing policy."
+        },
+        "storageAccountKey": {
+          "type": "string",
+          "description": "The primary storage account key of the Azure SQL Server auditing policy."
+        },
+        "storageAccountSecondaryKey": {
+          "type": "string",
+          "description": "The secondary key of the storage account of the Azure SQL Server auditing policy."
+        },
+        "storageTableEndpoint": {
+          "type": "string",
+          "description": "The storage table endpoint of the Azure SQL Server auditing policy."
+        },
+        "storageAccountResourceGroupName": {
+          "type": "string",
+          "description": "The resource group of the storage account of the Azure SQL Server auditing policy."
+        },
+        "storageAccountSubscriptionId": {
+          "type": "string",
+          "description": "The subscription Id of the storage account of the Azure SQL Server auditing policy."
+        },
+        "retentionDays": {
+          "type": "string",
+          "description": "The retention in days of the Azure SQL Database auditing policy."
+        },
+        "auditLogsTableName": {
+          "type": "string",
+          "description": "The name of the table audit logs are written to in the Azure SQL Database auditing policy."
+        },
+        "fullAuditLogsTableName": {
+          "type": "string",
+          "description": "The full name of the table audit logs are written to in the Azure SQL Database auditing policy."
+        }
+      },
+      "description": "Represents the properties of an Azure SQL auditing policy."
+    },
+    "ServerAuditingPolicy": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ServerAuditingPolicyProperties",
+          "description": "Represents the properties of the resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/SubResource"
+        }
+      ],
+      "description": "Represents an Azure SQL Server auditing policy."
+    },
+    "SubResource": {
+      "description": "Subresource properties",
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "format": "uuid",
```

Comment:
remove this, this is for consistency with other clients, not to have subscription id as GUID.